biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java

```diff
@@ -1458,6 +1458,10 @@ public boolean doNotCopy(String v) {
 		return getDoNotCopy().matcher(v).matches();
 	}
 
+	public boolean doNotCopy(File from) {
+		return doNotCopy(from.getName()) || doNotCopy(from.getAbsolutePath());
```

Comment:
There must be places in bnd where file names must handle windows. Perhaps @pkriens can point you to some utility classes to help here.